src/main/java/com/gmail/goosius/siegewar/playeractions/InvadeTown.java

```diff
@@ -121,7 +124,7 @@ private static void captureTown(Siege siege, Nation attackingNation, Town defend
 				attackingNation.getFormattedName()
 			));
 		}
-		if(nationDefeated) {
+		if (nationDefeated) {
```

Comment:
This file's changes to the spacing (lines 108, 113, 127) is something I prefer not to see, the end result is that it chops up the git blame. I'm fine when an block of code changes and someone re-formats things then. 

The issue is that goosius' IDE will never put a space in there, yours and mine always will. 

Ideally your IDE shouldn't be auto-formatting anything that wasn't written by yourself.